UI/index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Profit!</title>
+    <link rel="stylesheet" href="style.css">
+    <script>document.write('<script src="http://' + (location.host || 'localhost').split(':')[0] + ':35729/livereload.js?snipver=1"></' + 'script>')</script>
+</head>
+<body>
+    <div id="header">
+        <h1 class="profit_logo">Profit!</h1>
+        <div id="user_info">
+            <h2 id="username">username</h1>
+            <h2 id="sign_in_out">Sign out</h1>
+        </div>
+    </div>
+
+
+    <div id="filters">
+        <div id="date_filter">
+            <p>since</p>
+            <p class="blue_button">dd.mm.yyyy</p>
+            <p>till</p>
+            <p class="blue_button">dd.mm.yyyy</p>
+        </div>
+        <div id="hashtags_and_vendor">
+            <p>hashtags:</p>
+            <p class="blue_button">#hashtag1</p>
```

Comment:
The same: use input instead of p. Your UI should not only look great, but be useful as well.